src/components/standup/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { FC, useState, useEffect } from 'react';
+import moment from 'moment';
+import styles from '@/components/standup/standupContainer.module.scss';
+
+import {
+    useAddStandupMutation,
+    useUserStandupDetailsQuery,
+} from '@/app/services/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import FormInputComponent from './FormInputComponent';
+import { standupUpdateType } from '@/types/standup.type';
+import { getTotalMissedUpdate } from '@/utils/getTotalMissedUpdate';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+
+const StandUpContainer: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
```

Comment:
should we use the state for just style changes?

Reply:
we are not using state here just to change style but also we are using state here to enable or disable the button and on the basis of that we are changing style of a button, so I think we can use state here.